source/index.js

```diff
@@ -76,11 +72,9 @@ styles.visible = {
 	}
 };
 
-for (const model of Object.keys(ansiStyles.color.ansi)) {
-	if (skipModels.has(model)) {
-		continue;
-	}
+const usedModels = ['rgb', 'hsl', 'hsv', 'hwb', 'cmyk', 'xyz', 'lab', 'lch', 'hex', 'keyword', 'ansi', 'ansi256', 'hcg', 'apple'];
```

Comment:
There are models here we don't officially support. Only include the list in the docs: https://github.com/chalk/chalk#256-and-truecolor-color-support

Reply:
```suggestion
const usedModels = ['rgb', 'hex', 'keyword', 'hsl', 'hsv', 'hwb', 'ansi', 'ansi256'];
```

Reply:
> Note that applying this will probably cause these models to become undefined:

Which is fine as they're not documented.

Reply:
Just noticed HCG should be supported (in a separate PR, though). Opened #360.